scripts/verify_fideslang_2_data_migration.py

```diff
@@ -206,47 +207,47 @@ def create_outdated_objects() -> None:
     )
 
     # Create Privacy Notice
-    response = requests.post(
-        url=f"{SERVER_URL}/api/v1/privacy-notice",
-        headers=AUTH_HEADER,
-        allow_redirects=True,
-        data=json.dumps([old_notice.dict()]),
-    )
-    assert response.ok, f"Failed to Create Privacy Notice: {response.text}"
-    global PRIVACY_NOTICE_ID  # I'm so sorry
-    PRIVACY_NOTICE_ID = response.json()[0]["id"]  # Please forgive me
```

Comment:
This feels unintentional, probably want to uncomment these again?

Reply:
This is intentional, those models changed which breaks the script, and I don't think it is worth taking the time to update it now since this isn't where we had issues

Reply:
:+1